tuna/dbBase/base_class.py

```diff
@@ -34,23 +36,28 @@
 class BASE():
   """Base class for our own common functionalities among tables"""
 
-  __table_args__ = {'mysql_engine': 'InnoDB'}
-  __mapper_args__ = {'always_refresh': True}
+  __table__: Column = None
+  __table_args__: Dict[str, str] = {'mysql_engine': 'InnoDB'}
+  __mapper_args__: Dict[str, bool] = {'always_refresh': True}
 
-  id = Column(Integer, primary_key=True)
-  insert_ts = Column(DateTime, nullable=False, server_default=sqla_func.now())
-  update_ts = Column(
+  id: Column = Column(Integer, primary_key=True)
+  insert_ts: Column = Column(DateTime,
+                             nullable=False,
+                             server_default=sqla_func.now())
+  update_ts: Column = Column(
```

Comment:
If mypy doesn't complain about this I think it's better to leave the line as is, because the type is clearly shown by the constructor.